### PR TITLE
Fix #5 - Security Patches. Fixed all npm audit issues.

### LINKED ISSUE #5: Vulnerability : Uninitialized Memory Exposure in bl <0.9.5 || 1.0.0 package
Hi,
there is a vulnerability in the bl package, take a look : https://snyk.io/vuln/npm:bl:20160119

### PR DESCRIPTION
**Pull request by micksatana**
Security Patches. Fixed all npm audit issues.

[x] Upgrade dependencies.
  - There're changes in how to use mime types and how to lookup types.
[x] `npm test` all passed successfully.

### CODE REVIEW

**Review by chiendv — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Logs
```
Comment (mikeal) [inline .gitignore]: Can we get this file out of the PR. It's a little larger than most standard .gitignore files.

File: index.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 var stream = require('stream')
   , mime = require('mime')
+  , mimeTypes = require('mime-types')
```
Comment (mikeal) [inline index.js]: If we're moving to this new module I'd like to see us remove the old one entirely.

File: tests/test-location.js [hunk 3]
```diff
@@ -26,12 +26,12 @@ tape.test('testing new view mapping, convenience functions, and member functions
     t.plan(2)
     q.series([
     function(lib) {
-        server.listen(8082,function(){
+        server.listen(8084,function(){
```
Comment (mikeal) [inline tests/test-location.js]: Is there a reason this needed to change? My guess is that you changed this locally to get a test working after a prior one had been left hanging and it ended up in the pull request :)

**Review by mikeal — CHANGES_REQUESTED** [review]

**Comment by mikeal** [comment]
Thanks! I made one more adjustment, removing package-lock.json, before merging :)

**Pull request merged**